pkg/cluster/task/mkdir.go

```diff
@@ -31,7 +31,7 @@ type Mkdir struct {
 func (m *Mkdir) Execute(ctx *Context) error {
 	exec, found := ctx.GetExecutor(m.host)
 	if !found {
-		return ErrNoExecutor
+		panic(ErrNoExecutor)
```

Comment:
Why panic?

Reply:
Same with https://github.com/pingcap/tiup/blob/master/pkg/cluster/task/env_init.go#L54

I think it's easier to debug...

Reply:
Both of them should be changed to Err, panic is always a bad idea in our scenarios and we shouldn't display the stack trace to users.

Reply:
Ok

Reply:
Better to revert this part.